component/azstorage/azstorage.go

```diff
@@ -63,6 +63,8 @@ const compName = "azstorage"
 //Verification to check satisfaction criteria with Component Interface
 var _ internal.Component = &AzStorage{}
 
+var AzStatsCollector *internal.StatsCollector
```

Comment:
Do we need this to be public and a global var, it can be part of the azstorage object structure.